implants/lib/eldritch/src/pivot/ssh_password_spray_impl.rs

```diff
@@ -1,5 +1,12 @@
 use anyhow::Result;
 
-pub fn ssh_password_spray(_targets: Vec<String>, _port: i32, _credentials: Vec<String>, _keys: Vec<String>, _command: String, _shell_path: String) -> Result<String> {
+pub fn ssh_password_spray(
+    _targets: Vec<String>,
+    _port: i32,
+    _credentials: Vec<String>,
+    _keys: Vec<String>,
+    _command: String,
+    _shell_path: String,
+) -> Result<String> {
```

Comment:
I think in other function defs the args have all been tabbed in one col which is prettier.